otx/cli/builder/builder.py

```diff
@@ -184,92 +187,102 @@ def build_task_config(
         train_type_rel_path = ""
         if train_type != "incremental":
             train_type_rel_path = train_type
-        model_dir = os.path.join(os.path.abspath(template_dir), train_type_rel_path)
-        if not os.path.exists(model_dir):
+        model_dir = template_dir.absolute().joinpath(train_type_rel_path)
+        if not model_dir.exists():
             raise ValueError(f"[otx build] {train_type} is not a type supported by OTX {task_type}")
-        train_type_dir = os.path.join(workspace_path, train_type_rel_path)
-        os.makedirs(train_type_dir, exist_ok=True)
+        train_type_dir = workspace_path.joinpath(train_type_rel_path)
+        Path.mkdir(train_type_dir, exist_ok=True)
```

Comment:
[Optional] : Nice thing with pathlib is that it's super flexible, so even the following is possible 
```suggestion
        train_type_dir = workspace_path / train_type_rel_path
        train_type_dir.mkdir(exist_ok=True)
```

Reply:
First time using it, but it looks good. thank you!